test-cases/longevity/longevity-10gb-3h.yaml

```diff
@@ -8,8 +8,9 @@ n_monitor_nodes: 1
 
 instance_type_db: 'i3.4xlarge'
 
-nemesis_class_name: 'ChaosMonkey'
-nemesis_interval: 5
+nemesis_class_name: 'SisyphusMonkey'
```

Comment:
are you sure you want to change the 3h scenario?

Reply:
Yes, it's the closest thing we have to "sanity" and the 4h is more robust at the moment.